configure-time-zone.md

```diff
@@ -42,14 +38,12 @@ To set the format of the value of the `time_zone`:
 - The value can be given as a string indicating an offset from UTC, such as '+10:00' or '-6:00'.
 - The value can be given as a named time zone, such as 'Europe/Helsinki', 'US/Eastern', or 'MET'.
 
-The current session time zone setting affects the display and storage of time values that are zone-sensitive. This includes the values displayed by functions such as `NOW()` or `CURTIME()`.
+The current session time zone setting affects the display and storage of time values that are zone-sensitive. This includes the values displayed by functions such as [`NOW()`](/functions-and-operators/date-and-time-functions.md) or `CURTIME()`. The `UTC_TIMESTAMP()` function can be used to get a timestamp based on UTC, which can be useful to avoid time zone specific issues. The `CONVERT_TZ()` function can be used to convert between timezones.
```

Comment:
I would not say that it affects the "storage" of time values, but rather the interpretation and display of TIMESTAMP column type values, as well as some of the mentioned time and date functions. DATETIME/TIME stores the time as given (no conversion) and TIMESTAMP stores the time in UTC (conversion before storing and when retrieving).